packages/emotion-primitives/src/splitProps.js

```diff
@@ -0,0 +1,94 @@
+import { isValidStyleProp } from './props'
+
+const forwardableProps = {
```

Comment:
1. by converting it to object we can do a simple lookup (`if (forwardableProps[propName])`) instead of parsing this string with regexp each time this was looping
2. also I have found some duplicate properties in this string, so I've just smply removed them